man/systemd.netdev.xml

```diff
@@ -571,6 +571,16 @@
           <xi:include href="version-info.xml" xpointer="v248"/>
         </listitem>
       </varlistentry>
+     <varlistentry>
+        <term><varname>BroadcastThresholdLimit=</varname></term>
+        <listitem>
+          <para>Specifies the broadcast queueing threshold on macvlan devices. This controls which multicast
+          packets will be processed in a workqueue instead of inline. Setting this to -1 disables broadcast
```

Comment:
Please do not use `-1` for disabling the feature. Please use e.g. `"no"` or so to disable the feature.